.github/workflows/build-play-smoke-dist.yaml

```
@@ -0,0 +1,28 @@
+name: Build Play smoke test distribution
+
+on:
+  push:
+    paths:
+      - 'smoke-tests/play/**'
+    branches: 'master'
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 11 for running Gradle
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+
+      - name: Cache gradle dependencies
+        uses: burrunan/gradle-cache-action@v1
+        with:
+          job-id: jdk11
+
+      - name: Login to GitHub Package Registry
+        run: echo ${{ secrets.GITHUB_TOKEN }} | docker login docker.pkg.github.com -u ${{ github.repository }} --password-stdin
+
+      - name: Build Docker Image
+        run: cd smoke-tests/play && ../../gradlew jib
```

now that this dir has gradle?
```suggestion
        run: cd smoke-tests/play && ./gradlew jib
```